jobs/release.Jenkinsfile

```diff
@@ -275,6 +282,39 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
             }
         }
 
+        if (brew_profile) {
+            stage('Brew Upload') {
+                def tag = pipecfg.streams[params.STREAM].brew_tag
+                for (arch in basearches) {
+                    def state = false
+                    def version = (params.VERSION).replaceAll("-", ".")
+                    def nvr = "rhcos-${arch}-${version}-0"
+                    state = shwrapCapture("""
+                    coreos-assembler koji-upload search \
+                        --nvr ${nvr} \
+                        --keytab "/run/kubernetes/secrets/brew-keytab/brew.keytab" \
+                        --owner ${brew_principal} \
+                        --profile ${brew_profile}
+                    """)
```

Comment:
what gets returned here if the build doesn't exist?

Reply:
I mean, the build state itself goes to the output and the return 0 doesn't.

Reply:
> However, the 'shwrapCapture' needs to have something as output.

What if we made the output was just an empty string? Would that be OK?